draft-ietf-mops-streaming-opcons.md

```diff
@@ -483,35 +485,39 @@ Here are a few common resolutions used for video content, with typical ranges of
 
-* Media servers also typically rely on transport protocols with capacity-seeking congestion controllers that probe for bandwidth and adjust the sending rate based on transport mechanisms. This is described in greater detail in {{sec-trans}}.
+* Media servers also typically rely on transport protocols with capacity-seeking congestion controllers that probe for available path bandwidth and adjust the media segment sending rate based on transport mechanisms. This is described in greater detail in {{sec-trans}}.
 
 The result is that these two (potentially competing) "helpful" mechanisms each respond to the same bottleneck with no coordination between themselves, so that each is unaware of actions taken by the other, and this can result in QoE for users that is significantly lower than what could have been achieved.
 
-One might wonder why media servers and transport protocols are each blissfully ignorant of what the other is doing, and there are multiple reasons for that. However, one reason is that media servers may be implemented to run under a general-purpose operating system that typically has its transport protocols implemented in the operating system kernel, making decisions that the media server never knows about.
+One might wonder why media servers and transport protocols are each so blissfully ignorant of what the other is doing, and there are multiple reasons for that. One reason is that media servers are often implemented as applications executing in user space, relying on a general-purpose operating system that typically has its transport protocols implemented in the operating system kernel, making decisions that the media server never knows about.
 
 In one example, if a media server overestimates the available bandwidth to the media player,
 
 * the transport protocol detects loss due to congestion and reduces its sending window size per round trip,
 * the media server adapts to application-level feedback from the media player and reduces its own sending rate,
-* the transport protocol sends media at the new, lower rate and confirms that this new, lower rate is "safe" because no transport-level loss is occurring, but
+* the transport protocol sends media segments at the new, lower rate and confirms that this new, lower rate is "safe" because no transport-level loss is occurring, but
 * because the media server continues to send at the new, lower rate, the transport protocol's maximum sending rate is now limited by the amount of information the media server queues for transmission, so
-* the transport protocol cannot probe for available path bandwidth by sending at a higher rate.
+* the transport protocol cannot probe for available path bandwidth by sending at a higher rate, until the media receiver signals the media server that the media server can increase its media segment sending rate.
```

Comment:
I think the new clause is confusing (the signal generally comes via requesting a different url, not exactly by signaling an increased segment send rate).  Suggest maybe instead:

"until the media receiver requests segments that buffer enough data for the transport to perform the probing"

Reply:
Will be in committed in branch issue-155a.